requirements.txt

```diff
@@ -0,0 +1,5 @@
+Flask==2.3.0
```

Comment:
I'd like you to use Flask version 3.0.0
You can use whatever Werkzeug < 3.0.0 that works but the later the better. Ideally 2.3.7 unless there is a issue with it.

If there are issues, pop the errors in Slack.